canonical/ubuntu.com
# K8s docs changes for 1.28

## Done

- Added 1.28 components, upgrading and release notes page
- added main release notes
- updated metallb docs
- updated navigation
-  updated release notes for several bugfix updates of prior releases 

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- A reminder - ALL these changes have already been reviewed and approved by the K8s team - we aren't expecting you to read all the content, just check there is nothing obvious wrong with the new/changed pages e.g. misrendering.
